### ISSUE TITLE
Mac is hitting issue with FD

### ISSUE THREAD

**Issue by tomelliot16** [nhurel/terraspec#1]
```
runtime: netpoll: break fd ready for -2
fatal error: runtime: netpoll: break fd ready for something unexpected

runtime stack:
runtime.throw(0x23ac12c, 0x39)
	/usr/local/go/src/runtime/panic.go:1116 +0x72
runtime.netpoll(0x205bc9, 0x622a3021a201)
	/usr/local/go/src/runtime/netpoll_kqueue.go:140 +0x30a
runtime.findrunnable(0xc00004e800, 0x0)
	/usr/local/go/src/runtime/proc.go:2329 +0x72b
runtime.schedule()
	/usr/local/go/src/runtime/proc.go:2526 +0x2fc
runtime.park_m(0xc000000c00)
	/usr/local/go/src/runtime/proc.go:2696 +0x9d
runtime.mcall(0x80000)
	/usr/local/go/src/runtime/asm_amd64.s:318 +0x5b

goroutine 1 [runnable, locked to thread]:
github.com/ChrisTrenkamp/goxpath/parser.init()
	/go/pkg/mod/github.com/!chris!trenkamp/goxpath@v0.0.0-20170922090931-c385f95c6022/parser/ast.go:19

goroutine 18 [select]:
go.opencensus.io/stats/view.(*worker).start(0xc00008d900)
	/go/pkg/mod/go.opencensus.io@v0.22.0/stats/view/worker.go:154 +0x100
created by go.opencensus.io/stats/view.init.0
	/go/pkg/mod/go.opencensus.io@v0.22.0/stats/view/worker.go:32 +0x57
```

**Comment by nhurel** [comment]
I build and provide mac binary but i'm not able to test them myself. Can you give more info about your issue : 
- Did you run the binary or run from sources (via `go run..`) ?
- What terraform config and spec were you trying to test ?

**Comment by tomelliot16** [comment]
I was able to fix the issue by updating to terraform 0.13 I have a patch in flight that fixes the issue. Will have it up soon

**Comment by tomelliot16** [comment]
Looks like this is fixed now
